tests/apollo/test_skvbc_fast_path.py

```diff
@@ -105,6 +105,7 @@ def setUp(self):
 
     @with_trio
     @with_bft_network(start_replica_cmd,
+                      num_clients = 4,
```

Comment:
nit - according to Python convention (PEP8), kwargs do not have spacing around the ```=``` character. We should have `num_clients=4`